fighthealthinsurance/ml/ml_router.py

```
@@ -294,37 +294,48 @@ def add_model_name(model):
             List of RemoteModelLike models suitable for QA tasks
         """
-        if not use_external:
-            return []
+        forced = self._get_forced_models("for QA", use_external=use_external)
+        if forced:
+            return forced
 
-        # Add Llama Scout model if available
-        if "meta-llama/Llama-4-Scout-17B-16E-Instruct" in self.models_by_name:
-            return self.cheapest("meta-llama/Llama-4-Scout-17B-16E-Instruct")
+        models: list[RemoteModelLike] = []
```

Ruff (0.15.7)

[error] `RemoteModelLike` may be undefined, or defined from star imports

(F405)